crc/scripts/log.py

```diff
@@ -0,0 +1,64 @@
+from crc import session
+from crc.api.common import ApiError
+from crc.models.task_log import TaskLogModel, TaskLogModelSchema
+from crc.scripts.script import Script
+
+
+class TaskLog(Script):
```

Comment:
Things stay a lot clearer if we name this class Log, rather than TaskLog

Reply:
That's funny. I originally wrote it as Log, but since it could only be used from a ScriptTask, I renamed it.